### DISCUSSION TITLE
How to prevent image jumping, when new element added to FlashList

### DISCUSSION THREAD

**Discussion by szbenceg** [Shopify/flash-list#740] [Q&A]
I have a similar screen view to this example that you created. [merge request](https://github.com/Shopify/flash-list/pull/266) And when I send a message the image jumps down, and after back. 

This behavior can also be seen in [the video](https://user-images.githubusercontent.com/477473/160874898-2ef894dc-c882-4411-8146-67e163d27c35.mp4).
![image](https://user-images.githubusercontent.com/40304615/213583478-6b3770ac-3738-491e-9ee0-4f0a0711ff4d.png)
And after one second.
And a few milliseconds later, it jumps back to the correct place:
![image](https://user-images.githubusercontent.com/40304615/213583677-2e39954d-ff0f-4d3c-862d-92c3eb65335b.png)

If the image is cached everything works fine, but without caching the image how can I solve this behavior?
Is it because every item rerenders in the list when a new message is added? (Sorry if it is not an appropriate question, I am just trying to understand how FlashList works)

Thank you for your help in advance. :)

**Comment by siddharthkr** [comment]
Hi @szbenceg,

did you got any solution for this ? I am facing the same issue.

**Reply by szbenceg** [reply]
Not yet unfortunately.

**Reply by szbenceg** [reply]
Did you find any solution?
